### PR TITLE
add a basic readme

### PR DESCRIPTION
**Pull request by jakemac53**
fixes https://github.com/dart-lang/build/issues/65.

Assumes current phases design in https://github.com/dart-lang/build/issues/67.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,224 @@
-# build [![Build Status](https://travis-ci.org/dart-lang/build.svg?branch=master)](https://travis-ci.org/dart-lang/build) [![Coverage Status](https://coveralls.io/repos/dart-lang/build/badge.svg?branch=master)](https://coveralls.io/r/dart-lang/build)
+# Build [![Build Status](https://travis-ci.org/dart-lang/build.svg?branch=master)](https://travis-ci.org/dart-lang/build) [![Coverage Status](https://coveralls.io/repos/dart-lang/build/badge.svg?branch=master)](https://coveralls.io/r/dart-lang/build)
```
Comment (kevmoo) [inline README.md]: Skip the title entirely. Redundant in all contexts.

**Comment by kevmoo** [comment]
Needs to be updated for latest hotness?

**Comment by jakemac53** [comment]
removed the title, the rest should already be updated

**Comment by kevmoo** [comment]
LGTM – please rebase :smile:

**Pull request merged**